examples/seismic/tutorials/Gradient_full_marmousi.py

```diff
@@ -0,0 +1,163 @@
+r'''
+Memory profiling is performed to compute the memory usage when computing the gradient computation
+using the full time axis.
+
+Memory-profiler run as: mprof run Gradient_full_marmousi.py; mprof plot
+'''
+
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.ndimage import gaussian_filter
+import scipy
+from memory_profiler import memory_usage
```

Comment:
imports from stdlib at the very top

then blank line

then imports from third parties (eg scipy)

then blank line

then examples imports
then devito imports

Reply:
Fixed